### DISCUSSION TITLE
Safe default routing for hikers

### DISCUSSION THREAD

**Discussion by wolfbert** [osmandapp/OsmAnd#11309] [Ideas]
I'd like to venture some ideas with regard to routing for hikers. I'm from Austria, and there is an increasing number of (inexperienced) hikers who just follow their navigation app, often using less than recommended routes, and ending up calling mountain rescue (if they still can).

Naturally, there's only so much the routing engine can do, and it depends on the map data as well. But every little bit may help, and most routing apps just don't care. I've had a brief look at routing.xml, maybe a separate profile for hiking would make sense.

Tags that should be considered (if present) are e.g.

- [sac_scale](https://wiki.openstreetmap.org/wiki/Key:sac_scale) Only "hiking" should be considered by default (this could be configurable in the profile for more experienced hikers).

- [trail_visibility](https://wiki.openstreetmap.org/wiki/Key:trail_visibility) Only paths with values "excellent" and "good" should be considered by default (this could be configurable in the profile for more experienced hikers).

- [informal=yes](https://wiki.openstreetmap.org/wiki/Key%3Ainformal)  A path that has not been created deliberately. Should be avoided by default (this could be configurable in the profile for more experienced hikers).

- [access=discouraged](https://wiki.openstreetmap.org/wiki/Key%3Aaccess) Not in widespread use, should obviously not be routed over.

In addition, rules for pedestrians would apply. I realize that this is to protect people from themselves, but at least chances are reduced that they get themselves into trouble without making a conscious decision.

Any thoughts are appreciated.

Best regards
Wolfgang

**Comment by barteis** [comment]
Interesting idea, if not for the default hiking profile, at least for a "safe/mainstream" variation. One possible problem is that from what I've seen in California, none if these four tags is widely used. Obviously, a missing tag shouldn't be a barrier to entry.

**Comment by tommycrock** [comment]
I agree with Bart. I get the principal but it feels like one to apply to
the standard "on foot" mode, but I'd be disappointed to find out I'd
enabled a hiking mode and some paths were off-limits, especially if it was
not obvious why or how to change it.

Maybe a mountaineering mode would enable everything?

Thank you for pointing out informal=yes. I think I'll be using that a lot.
But I don't think as documented it's something to not route over in hiking
mode as it includes desire lines in parks, etc. But this may be a good one
to avoid in the basic on foot mode?

The other point made by Bart is important. Unfortunately the sac scale and
trail visibility tags are only very applicable in the Alps and maybe a few
other high mountain areas as far as I'm aware. We could really do with some
tags that made it clear that this one will be fine with elderly relatives,
for this the route isn't very obvious, for another you should expect to
scramble, etc.

**Reply by barteis** [reply]
I was also unaware of informal=yes.  See also: https://wiki.openstreetmap.org/wiki/Proposed_features/Social_path

**Comment by wolfbert** [comment]
Thank you for your thoughts.

As for how/where this functionality could be integrated in the UI, I'd go with whatever is most useful and convenient. I'd make it the default nonetheless, because anyone using a navigation device in the mountains should at least spend a few minutes to set it up. If that is asking too much, that person shouldn't go off the beaten path anyway.

Admittedly, the tags described are mostly used in the Alps, and even so there have been discussions. I've found an additional tag [rtsa_scale](https://wiki.openstreetmap.org/wiki/Key:rtsa_scalel) describing the difficulty of mountain passes in former USSR, and other schemes may be in use elsewhere. That said, tagging and tagging schemes are beyond the scope of this discussion. Osmand can only work with the tags in use at the moment.

**Reply by barteis** [reply]
Around here in the San Francisco area and in the Sierra, a simple "avoid unnamed paths" option would do a lot to keep neophytes safer and the rest of us where we generally prefer to be.
